lib/npm-auditer.js

```diff
@@ -6,6 +6,9 @@
 const { runProgram, reportAudit } = require('./common');
 const Model = require('./Model');
 
+const RETRY_ERROR_MSG =
+  'npm ERR! audit Your configured registry (https://registry.npmjs.org/) does not support audit requests.';
```

Comment:
Since we use the `--registry` flag, this may not be the correct error message since `https://registry.npmjs.org/` may be incorrect. Perhaps simplify it to `'does not support audit requests.'` or pass in the `--registry` variable and make `RETRY_ERROR_MSG` a function. If the NPM error message changes in the future this is brittle, but it is undoubtedly better than the existing setup.